public/js/components/Graph.js

```diff
@@ -11,29 +11,31 @@ export default class Graph extends React.Component {
     }
 
     componentDidMount() {
-        // TODO: Use actuial data
-        fetch(`keywords/trump`)
-            .then(res => res.json())
-            .then((dataPoints) => {
-                dataPoints = dataPoints
-                    .sort((a, b) => a.times - b.times)
+        fetch(`keywords/top`)
```

Comment:
@JackyChiu Should this be done by an action? Or since the 'GraphActions' would only have one purpose, its best to do it here?